CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
+* add lambda syntax to grammar ([#6868](https://github.com/confluentinc/ksql/pull/6868)) ([dd3f365](https://github.com/confluentinc/ksql/commit/dd3f365eba725c77cca63f9196dac174394ecb58))
+* adding TypeContext for Lambdas ([#6989](https://github.com/confluentinc/ksql/pull/6989)) ([1538b3a](https://github.com/confluentinc/ksql/commit/1538b3ad7975adbc1ad0a452a7538ad7f3f748ba))
+* Adds ability to bypass cache for pull queries ([#6891](https://github.com/confluentinc/ksql/pull/6891)) ([4b3bc96](https://github.com/confluentinc/ksql/commit/4b3bc96830fbea8051cfaf4e9d27c4399032813f))
+* Adds table scans to pull queries ([#6726](https://github.com/confluentinc/ksql/pull/6726)) ([ae14fca](https://github.com/confluentinc/ksql/commit/ae14fca76089b85653de15003aead0dae4d2a5de))
+* Allows pull queries with generic WHERE clauses ([#6939](https://github.com/confluentinc/ksql/pull/6939)) ([c3fe8a1](https://github.com/confluentinc/ksql/commit/c3fe8a1aaf17c40f1205374f13a633ca6ebdbe19))
+* Move WHERE clause handling for pull queries into logical plan ([#6814](https://github.com/confluentinc/ksql/pull/6814)) ([fa6758a](https://github.com/confluentinc/ksql/commit/fa6758af2f46594bfdf8f18a6d59a9b40e62e1d0))
+* new and initialize commands for migration tool ([#6988](https://github.com/confluentinc/ksql/pull/6988)) ([8bdb09a](https://github.com/confluentinc/ksql/commit/8bdb09a2751365966cf5a4a1436276d7a3e0d9e1))
+* add filter lambda udf ([#7075](https://github.com/confluentinc/ksql/pull/7075)) ([d6529a3](https://github.com/confluentinc/ksql/commit/d6529a32d4d280a0b8c66c56228dedfc45a2bf03))
+* add maven wrapper ([#7307](https://github.com/confluentinc/ksql/pull/7307)) ([d0ab425](https://github.com/confluentinc/ksql/commit/d0ab4253d243d11783b8f81fba20fb7fae77038a))
+* Adds max concurrent pull queries to limit effects of table scans ([#7188](https://github.com/confluentinc/ksql/pull/7188)) ([55f5403](https://github.com/confluentinc/ksql/commit/55f5403eab223ca727ca5a6d4efb10b06b1dd97b))
+* CLI should fail for unsupported server version ([#7097](https://github.com/confluentinc/ksql/pull/7097)) ([a0745b9](https://github.com/confluentinc/ksql/commit/a0745b9e1a855be5be3c4cf5c2ffc7e384d29105))
+* **migrations:** enable insert values and create connector for migrations tool ([#7161](https://github.com/confluentinc/ksql/pull/7161)) ([2c614cd](https://github.com/confluentinc/ksql/commit/2c614cd80aeae81c71107de16fbf2649ce222b1c))
+* **migrations:** enable set/unset statements for migration tool ([#7190](https://github.com/confluentinc/ksql/pull/7190)) ([4151bae](https://github.com/confluentinc/ksql/commit/4151bae93632b9b2ed425db093051d4b58bffa7d))
+* add lambda feature flag ([#7148](https://github.com/confluentinc/ksql/pull/7148)) ([c8f745e](https://github.com/confluentinc/ksql/commit/c8f745e8ac8f034661234ec7334009df4b50aac1))

```

Comment:
Let's collect all the lambda-related commits together as well, rather than having them spread throughout the changelog.